Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
+
```

Comment:
In last few cargo.lock updates this part wasn't exist, maybe it should be removed until some decision on cargo.lock format version for rustc?